src/fmt.cc

```diff
@@ -0,0 +1,85 @@
+module;
+#ifndef FMT_MODULE_NAME
+#define FMT_MODULE_NAME fmt
+#endif
+// put all implementation-provided headers into the global module fragment
+// to prevent attachment to this module
+#include <cctype>
+#include <cerrno>
+#include <climits>
+#include <cmath>
+#include <cstdarg>
+#include <cstddef>
+#include <cstdint>
+#include <cstdio>
+#include <cstring>
+#include <ctime>
+#include <cwchar>
+
+#include <algorithm>
+#include <chrono>
+#include <exception>
+#include <functional>
+#include <initializer_list>
+#include <iterator>
+#include <limits>
+#include <locale>
+#include <memory>
+#include <ostream>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+#include <string_view>
```

Comment:
I don't think `<string_view>` is included anywhere in fmt.

Reply:
`core.h` line 263
All implementation-provided headers were grepped out of the sources.

Reply:
My bad, it's conditionally included from core.h